games/devtest/mods/unittests/player.lua

```diff
@@ -84,3 +84,23 @@ local function run_player_add_pos_tests(player)
 end
 unittests.register("test_player_add_pos", run_player_add_pos_tests, {player=true})
 
+--
+-- Hotbar selection clamp
+--
+local function run_player_hotbar_clamp_tests(player)
+	local inv = player:get_inventory()
+	local old_inv_size = inv:get_size("main")
+	local old_bar_size = player:hud_get_hotbar_itemcount()
+
+	inv:set_size("main", 5)
```

Comment:
not that it matters much but this potentially deletes items from the player's inventory, no?

Reply:
Fixed in f8ac49787